osmosis-1/osmosis-1.assetlist.json

```diff
@@ -1155,6 +1155,39 @@
         "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/fet.svg"
       },
       "coingecko_id": "fetch-ai"
+    },
+    {
+      "description": "The native token of Carbon",
+      "denom_units": [
+        {
+          "denom": "ibc/8FEFAE6AECF6E2A255585617F781F35A8D5709A545A804482A261C0C9548A9D3",
+          "exponent": 0,
+          "aliases": [
+            "swth"
+          ]
+        },
+        {
+          "denom": "dswth",
```

Comment:
`          "denom": "swth",
`
No Need to alias since that source denom, maybe confusion

Reply:
We are creating a new alias here so that the UI can show the correct 8 d.p. for human usage.

If we swap this, the unit denom won't match the source chain - do you think that is okay?

Reply:
The `ibc/xxxx` representation of source denom : `uatom` with 0 exponent should be the same between source <-> destination channel, also denom with exponent on-chain : 8 for your chain, need to be same on both.